### ISSUE TITLE
ResultHandler can not send event  {FFmpegKitExecuteCallbackEvent={create...}} on a null event sink.

### ISSUE THREAD

**Issue by husen-hn** [LeGoffMael/video_editor#51]
I have copied the code of the example app exactly and even merged the versions of all the packages, but I get this error when exporting. very strange
```bash
ResultHandler can not send event {FFmpegKitExecuteCallbackEvent={createTime=1636789797411, 
startTime=1636789797415, sessionId=2, type=1, command=-i /data/user/0/com.example.example/cache/file_picker
/VID_20211014_105136.mp4 -filter:v crop=153:221:0:98 -ss 0:00:00.000000 -to 0:00:01.096088 -y /data/user
/0/com.example.example/cache/VID_20211014_105136_1636789797408.mp4}} on a null event sink.
```
`video_editor: ^1.2.2`
`Flutter : 2.5.0`
`Android: 10 API 29`

**Comment by ghost** [comment]
I am facing the same issue

Flutter : 2.5.3
Mac os : 12.0.1
Dart: 2.14.4
Project Level Dart: 2.12.0

Also Using the latest version of this pub and my device has android 11 and Exporting progress is not starting and call back is not returning.

I've tried printing logs in OnProgress call back but it won't displayed.

**Comment by iamwaseemsh** [comment]
Yes i'm having same problem.
I ran the example, it worked fine, but then i tried to use video trimming code in my other project, then i got same error 
**ResultHandler can not send event {FFmpegKitExecuteCallbackEvent={createTime=1637178660202, startTime=1637178660214, sessionId=1, type=1, command=-i /data/user/0/com.bism.mylnk_io/cache/image_picker7250629880474435760.mp4 -filter:v crop=480:800:0:0 -ss 0:00:00.000000 -to 0:00:20.250000 -y /data/user/0/com.bism.mylnk_io/cache/image_picker7250629880474435760_1637178660041.mp4}} on a null event sink.**
So, i thought may be i made a mistake, then i directly used **VideoEditor**  class and pass the video file, again i got same issue, progress didnt even start, 
i also performed another test, on same project i was having problem i replaced that's project main.dart file with your example main.dart file, then tried to trim a video nothing worked then i stopped and started again the app then app worked,

**Comment by abhay-s-rawat** [comment]
I am also facing same issue , some says this happens when you are using firebase on background message functionality.

**Comment by LeGoffMael** [comment]
Like @abhay-s-rawat said you will get this error if you are implementing `FirebaseMessaging.onBackgroundMessage`

It is a ffmpegKit problem, it should be fixed in the next version (January 2022) https://github.com/tanersener/ffmpeg-kit/issues/209, or you could use the development.

This error probably doesn't occurs with `video_editor` < 2.0.1.

**Comment by husen-hn** [comment]
for now I just replace this with the current FFmpeg in packages's YAML file to resolve it.
```yaml
 ffmpeg_kit_flutter:
    git:
      url: https://github.com/shilangyu/ffmpeg-kit
      ref: development-flutter
      path: prebuilt/bundle-flutter-lts/default
```

**Comment by LeGoffMael** [comment]
should be fixed in the new 1.2.3
